ecommerce/admin.py

```diff
@@ -0,0 +1,75 @@
+"""
+Admin views for ecommerce models
+"""
+
+from django.contrib import admin
+
+from ecommerce.models import (
+    Line,
+    Order,
+    OrderAudit,
+    Receipt,
+)
+from bootcamp.utils import get_field_names
+
+
+class LineAdmin(admin.ModelAdmin):
+    """Admin for Line"""
+    model = Line
+
+    readonly_fields = get_field_names(Line)
+
+    def has_add_permission(self, request):
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        return False
+
+
+class OrderAdmin(admin.ModelAdmin):
+    """Admin for Order"""
+    model = Order
+
+    readonly_fields = [name for name in get_field_names(Order) if name != 'status']
+
```

Comment:
I've found it more useful to break out the fields in the list view, instead of having them in one string. It would also be useful to filter on status. Something like:

```
    list_display = ('id', 'user', 'status',)
    list_filter = ('status',)
```

It would also save extra steps if we could display some data from the line item here too, but I'm not sure if that's the klass title or the line description.